### PR TITLE
Generate new README

### PR DESCRIPTION
**Pull request by Bence1001**
### Checklist

- [x] I've read and accepted the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
- Requires _PATCH_ [version update](https://semver.org/)

### Context
* We are introducing a new README format for steps.

### Changes
* Added `generate-readme` workflow with `contrib_section` (see: `docs/contribution.md`).
* Generated new README.

### Decisions
* We've updated the step's name to `Deploy to iTunes Connect with Deliver` to differentiate it from the other iTunes Connect step.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,101 +1,106 @@
-# Deploy to iTunes Connect with Deliver
```
Comment (mateherber) [inline README.md:1]: It is weird that the title of the step doesn't contain the deliver.

Comment (mateherber) [reply README.md:1]: It does for the application loader

https://github.com/bitrise-steplib/steps-deploy-to-itunesconnect-application-loader/blob/master/step.yml#L1

Comment (Bence1001) [reply README.md:1]: I think we can change that without breaking anything, right?

**Review by mateherber — APPROVED** [review]

**Review by mateherber — APPROVED** [review]

**Comment by Bence1001** [comment]
We have to wait with restarting the build because we reached our upload limit to iTunes Connect 🙃

**Pull request merged**